### PR TITLE
Fix tutorial page card not translated

### LINKED ISSUE #838: Investigate why tutorial index page in not localized
Investigate why this page: https://substrate.dev/zh-CN/tutorials is not localized

### PR DESCRIPTION
**Pull request by jimmychu0807**
Close #838

### CODE REVIEW

**Comment by jimmychu0807** [comment]
Integrating Dan PR #836

File: docs/tutorials/visualize-node-metrics.md [hunk 1]
```diff
@@ -179,12 +179,19 @@ Hit `Save & Test` to ensure that you have the data source set correctly. Now you
...
+If you create your own, the
+[prometheus docs for grafana use](https://prometheus.io/docs/visualization/grafana/)
+may be helpful.
+
+<b>If you do create one, consider uploading it to the
 [community list of dashboards](https://grafana.com/grafana/dashboards) and letting
 the substrate builder community know it exists by listing in on
-[Awesome Substrate](https://github.com/substrate-developer-hub/awesome-substrate)!</b>
+[Awesome Substrate](https://github.com/substrate-developer-hub/awesome-substrate)!
+Here is ours on [the grafana public dashboards](https://grafana.com/grafana/dashboards/13759/)</b>
```
Comment (apopiak) [inline docs/tutorials/visualize-node-metrics.md:194]:
```suggestion
Here is ours on [the grafana public dashboards](https://grafana.com/grafana/dashboards/13759/).</b>
```

File: docs/tutorials/visualize-node-metrics.md [hunk 2]
```diff
@@ -198,7 +205,7 @@ the substrate builder community know it exists by listing in on
 
 - The Grafana dashboard configuration for the [Polkadot
   network](https://github.com/w3f/polkadot-dashboard).
-- [Grafana Template](https://grafana.com/grafana/dashboards/11784/) for a Substrate Node Template
+- [Grafana Template](https://grafana.com/grafana/dashboards/13759/) for a Substrate Node Template
```
Comment (apopiak) [inline docs/tutorials/visualize-node-metrics.md:208]: The points at the end of the line should probably be consistent

Comment (jimmychu0807) [reply docs/tutorials/visualize-node-metrics.md]: updated.

**Comment by apopiak** [comment]
LGTM, is there a way to see the render of this page?

**Review by apopiak — APPROVED** [review]

**Comment by jimmychu0807** [comment]
@apopiak 

1. One is to use the [local testing instruction here](https://github.com/substrate-developer-hub/substrate-developer-hub.github.io#local-testing)
2. Another is pushing to the `staging-source` branch. This will trigger the CI to build in the [staging site](https://devhub-maindocs.herokuapp.com/)

**Pull request merged**